MainApp/MainApp/Feature/Profile/ProfileLanding/ProfileLandingController.swift

```diff
@@ -11,20 +11,29 @@ import Component
 
 class ProfileLandingController: MasterViewController, ProfileLandingViewProtocol, UITableViewDelegate, UITableViewDataSource {
```

Comment:
datasource should be handled outside of view controller

Reply:
datasource moved to view model